documentation/userguide.md

```diff
@@ -231,62 +231,75 @@ This may, however, be a rather large file since the entire program's heap will b
 
 # Automatic Trace Upload
 
-The profiler can automatically upload all produced trace files to Teamscale or
-move them to a file system directory (e.g. a network share).
+The profiler can automatically upload all produced trace files to Teamscale,
+move them to a file system directory (e.g. a network share), ...
```

Comment:
Do we have the full list of upload alternatives somewhere? If so, can we reference it here? Should we give configuration examples for all options (Azure File Storage is currently missing)?

Reply:
I listed them explicitly here now. Please create a ticket for the Azure example. I think you should write that :-)

Reply:
#82